CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 ## 2.0.0-BETA2 (202x-xx-xx)
+* possible BC Break if final controller class was copied: Removed/replaced redundant argument `$firewallNames` from controllers. Use `$resourceOwnerMapLocator->getFirewallNames()` instead.
```

Comment:
If anyone copied final class it's not BC break, final means we don't support that, only composition which still is not hit here.

Let's write this as enhancement.